### PR TITLE
Bugfixes for 1.2.0

### PR DESCRIPTION
**Pull request by Safihre**
Mostly bug fixes and update of the year.

### CODE REVIEW

File: README.mkd [hunk 1]
```diff
@@ -28,6 +28,7 @@ Release Notes  -  SABnzbd 1.2.0
 - Dropped dependency on PyOpenSSL, now only requires cryptography package
```
Comment (shypike) [inline README.mkd]: I agree with Zoggy. Don't remove yet.

**Comment by Safihre** [comment]
@shypike 

- Skimming through SABnzbd.py I removed the ```RotatingFileHandler``` for Windows that was added long ago but never used and removed CherryPy config settings that don't exist anymore
- Updated all the Copright's already to 2017, just so we don't forget
- On shutdown/restart the notifier would send a 'Shutdown completed' notification to the Windows Icon, even though it was already gone.
- Updated the check if there's already a version of SABnzbd active to also include the the current `__version__` so it also works for people on `develop`.
- When there was a recursive unpacking in the destination folder, the path was not shortened on Windows causing UnRar to fail when extracting.

**Comment by war59312** [comment]
Typo on line 564.

> Explain erver optional tickbox

To:

> Explain server optional tickbox

Missed the s, whoops. ;)

**Comment by Safihre** [comment]
@war59312 Luckily just in the comment :)

**Comment by war59312** [comment]
Indeed ;)

**Pull request merged**